CRUDEFutures/CRUDE.swift

```diff
@@ -215,28 +214,7 @@ public struct CRUDE {
         return NSError(domain: issue, code: (network.response?.statusCode ?? -1), userInfo: debugInfo)
     }
 
-    private static var defaultLogger: CRUDELog = { type, network in
```

Comment:
Logging is very app-specific. If the client wants to log, they can easily use or modify the example code.